auth-api/src/auth_api/services/affidavit.py

```diff
@@ -119,7 +119,8 @@ def _modify_task(user):
 
                 # Send notification mail to staff review task
                 from auth_api.services import Org as OrgService  # pylint:disable=cyclic-import, import-outside-toplevel
-                OrgService.send_staff_review_account_reminder(relationship_id=org.id)
+                user = UserModel.find_by_jwt_token()
```

Comment:
This can be changed by injecting user_context to send_staff_review_account_reminder so that callers doesn't need to pass it.